uniqueness_management_system.py

```
@@ -41,6 +44,15 @@ class ConflictResolution(Enum):
     KEEP_EXISTING = "keep_existing"
     MANUAL_REVIEW = "manual_review"
 
+def get_created_by_info():
+    """Get the created_by string for records added by the guitar processor."""
+    try:
+        version = importlib.metadata.version('gtr-reg')
+        return f"guitar_processor_cli_v{version}"
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)